submissions/models.py

```diff
@@ -41,7 +55,7 @@ class StudentItem(models.Model):
 
     """
     # The anonymized Student ID that the XBlock sees, not their real ID.
-    student_id = models.CharField(max_length=255, blank=False, db_index=True)
+    student_id = AnonymizedUserIDField()
```

Comment:
Am I remembering correctly that this doesn't require any migration, because the underlying storage is the same.